### PR TITLE
Display related policies on policy group pages.

### PR DESCRIPTION
**Pull request by elliotcm**
These existed on the page before, and need restoring.

https://github.com/alphagov/rummager/pull/434 **MUST** be deployed before this.

Pre-requisites to things actually showing up (but not to this going out):
- [x] Merge and deploy https://github.com/alphagov/policy-publisher/pull/76 and do the things listed there

https://trello.com/c/pV7ExOoE/203-5-associate-policy-to-a-group

screenshot 2015-05-20 14 52 09

### CODE REVIEW

File: test/unit/policy_group_test.rb [hunk 1]
```diff
@@ -34,4 +37,24 @@ class PolicyGroupTest < ActiveSupport::TestCase
     Whitehall::PublishingApi.expects(:publish_async).with(policy_group).once
     policy_group.publish_to_publishing_api
   end
+
+  test "#published_policies should return the all policies" do
```
Comment (tekin) [inline test/unit/policy_group_test.rb]: "return the all policies"?

File: test/unit/policy_group_test.rb [hunk 2]
```diff
@@ -34,4 +37,24 @@ class PolicyGroupTest < ActiveSupport::TestCase
...
+    policy_group = create(:policy_group)
+    rummager_has_new_policies_for_every_type
+
+    all_policy_titles = [
```
Comment (tekin) [inline test/unit/policy_group_test.rb]: This is now duplicated in two sets of tests. Worth extracting to a helper to remove the duplication?

File: test/unit/policy_group_test.rb (hunk 1 above)
Comment (elliotcm) [inline test/unit/policy_group_test.rb]: This is how the cool kids are saying it now.

File: test/unit/policy_group_test.rb (hunk 2 above)
Comment (elliotcm) [inline test/unit/policy_group_test.rb]: Yeah probs.

**Comment by tekin** [comment]
:+1:

**Pull request merged**